src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/iOS/MediaElementRenderer.ios.cs

```diff
@@ -437,14 +446,11 @@ void AddRateObserver()
 		void AddStatusObserver()
 		{
 			DestroyStatusObserver();
-			statusObserver = playerItem.AddObserver("status", NSKeyValueObservingOptions.New, ObserveStatus);
+			statusObserver = playerItem?.AddObserver("status", NSKeyValueObservingOptions.New, ObserveStatus);
 		}
 
 		void AddPlayedToEndObserver()
 		{
-			DestroyPlayedToEndObserver();
-			playedToEndObserver =
-				NSNotificationCenter.DefaultCenter.AddObserver(AVPlayerItem.DidPlayToEndTimeNotification, PlayedToEnd);
```

Comment:
Good catch! That's actually a typo and I'll add it back.

I had temporarily removed this because I was trying to re-architect the class to limit the number of nullable fields, and had forgot to add it back in.